packages/grant-explorer/src/app/chainConfig.ts

```diff
@@ -0,0 +1,34 @@
+import {
```

Comment:
Note to self, we should really centralize this + contracts + tokens at some point

Reply:
Will be done soon.